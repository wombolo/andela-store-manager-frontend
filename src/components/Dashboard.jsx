import React, { Component } from 'react';

export default class Dashboard extends Component {
  render() {
    return (
      <div>
        <h1>My Dashboard</h1>
      </div>
    )
  }
}
