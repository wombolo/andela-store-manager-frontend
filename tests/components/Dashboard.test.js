import 'babel-polyfill'
import React from 'react';
import { shallow } from 'enzyme';

import Dashboard  from '../../src/components/Dashboard.jsx';

let wrapper;

describe('Dashboard Component', () => {
  it('should match snapshot', () => {
    wrapper = shallow(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
  });
});
