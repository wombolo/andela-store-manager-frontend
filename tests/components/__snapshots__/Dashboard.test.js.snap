// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Component should match snapshot 1`] = `
<div>
  <h1>
    My Dashboard
  </h1>
</div>
`;
